src/scss/_header.scss

```diff
@@ -1,28 +1,79 @@
 #header {
-	color: $opfront-white;
+  color: $opfront-white;
```

Comment:
You still have a 2 space indent here, but it was tabs before, which is weird. I tried reformatting the whole scss folder and there are actually 6 files that are not formatted properly... Can you reformat them all along with this PR please?